### PR TITLE
bemjxst: should keep `mods` in case of same block

### PR DESCRIPTION
**Pull request by miripiruni**
Fix for #254

### CODE REVIEW

File: lib/bemxjst/index.js [hunk 1]
```diff
@@ -354,7 +354,7 @@ BEMXJST.prototype.runOne = function runOne(json) {
     if (json.mods)
       context.mods = json.mods;
     else
-      context.mods = {};
+      context.mods = json.block === oldBlock ? oldMods : {};
```
Comment (zxqfox) [inline lib/bemxjst/index.js]: `else if (json.block === oldBlock) { context.mods = oldMods } else `?

Comment (zxqfox) [inline lib/bemxjst/index.js]: Ah, no.

just `else if (json.block !== oldBlock) { context.mods = {}; }`

Comment (miripiruni) [inline lib/bemxjst/index.js]: @zxqfox 👍

**Comment by miripiruni** [comment]
🆙

**Comment by zxqfox** [comment]
LGTM

**Comment by tadatuta** [comment]
👍

**Pull request merged**